cellprofiler/modules/shrinkobjectstopoints.py

```diff
@@ -0,0 +1,128 @@
+# coding=utf-8
+
+"""
+ShrinkObjectsToPoints
+======================
+
+**ShrinkObjectsToPoints** will transform a set of objects into a label image with single points
+representing each object. The location of each point corresponds to the centroid of the input objects.
+"""
+
+import cellprofiler.image
+import cellprofiler.module
+import cellprofiler.setting
+import numpy
+import skimage.measure
+
+
+class ShrinkObjectsToPoints(cellprofiler.module.ObjectProcessing):
+
```

Comment:
No space is necessary between the class/method/function declaration and the next line.

E.g., 
```python
class ShrinkObjectsToPoints(cellprofiler.module.ObjectProcessing):
    module_name = "ShrinkObjectsToPoints"
```

```python
def create_settings(self):
    super(ShrinkObjectsToPoints, self).create_settings()
```